src/olympia/core/utils.py

```diff
@@ -5,15 +5,32 @@
 import django
 
 
-def get_version_json():
-    contents = {}
+# Keys exempt from inspection in local images as they must be set.
+EXEMPT_REQUIRED_KEYS = ['target', 'version', 'source']
```

Comment:
I don't see where this is used independently - what is it for?

Reply:
Right you are.